ldi-core/ldes-client-v2/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/domain/valueobject/Ldes.java

```diff
@@ -0,0 +1,23 @@
+package ldes.client.treenodesupplier.domain.valueobject;
+
+import ldes.client.treenodefetcher.domain.valueobjects.TreeNodeRequest;
+import org.apache.jena.riot.Lang;
+
+public class Ldes {
```

Comment:
see my previous comment on the name of this object

Reply:
Frankly, I find this name to be most vague.. maybe an call this a LdesStartingNode ? Needs an extra thought/discussion